apps/data/io/parsers.py

```diff
@@ -62,8 +65,15 @@ def _to_entries(self, ignore_aliases):
             url = '{}{}'.format(self.root_url, feature['id'])
             aliases = []
 
+            if pandas.isna(feature['name']):
+                logger.warning(f'Invalid feature name for id={id}')
+                continue
+
             if not pandas.isna(feature['aliases']) and not ignore_aliases:
-                aliases = map(str, feature['aliases'].split('|'))
+                aliases = filter(
+                    lambda a: len(a) <= 100,
+                    map(str, feature['aliases'].split('|'))
+                )
```

Comment:
I would add a warning message when we ignore one or more aliases:

```python
splitted_aliases = feature['aliases'].split('|')

aliases = filter(
    lambda a: len(a) <= 100,
    map(str, splitted_aliases)
)

if len(aliases) != len(splitted_aliases):
    logger.warning('Ignored long aliases (100+ chars) in {}'.format(feature['aliases'])
```

Reply:
good call